src/main/java/micdoodle8/mods/galacticraft/core/client/gui/overlay/OverlayOxygenTanks.java

```diff
@@ -32,20 +31,19 @@ public static void renderOxygenTankIndicator(int heatLevel, int oxygenInTank1, i
         final int i = scaledresolution.getScaledWidth();
         final int j = scaledresolution.getScaledHeight();
         OverlayOxygenTanks.minecraft.entityRenderer.setupOverlayRendering();
-        GL11.glEnable(GL11.GL_BLEND);
-        GL11.glDisable(GL11.GL_DEPTH_TEST);
-        GL11.glDepthMask(false);
-        GL11.glBlendFunc(GL11.GL_SRC_ALPHA, GL11.GL_ONE_MINUS_SRC_ALPHA);
-        GL11.glColor4f(1.0F, 1.0F, 1.0F, 1.0F);
-        GL11.glDisable(GL11.GL_ALPHA_TEST);
+        GlStateManager.enableBlend();
+        GlStateManager.disableDepth();
+        GlStateManager.depthMask(false);
+        GlStateManager.blendFunc(770, 771);
```

Comment:
Using GlStateManager is a good change, but the so-called _magic constants_ should be reverted back to the values before your commit.

E.g. `770`-> `GL11.GL_SRC_ALPHA`